client/app/account/component/AccountInfo.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+// import Link from 'next/link'
+import { MdAccountCircle } from 'react-icons/md'
+import { Loading } from '@/_components/layout'
+import { colorMap } from '@/_consts/iconColor'
+import { useGetMeQuery } from '@/_hooks/api'
+
+const AccountInfo = () => {
+  const { data: user, isLoading } = useGetMeQuery()
+
+  if (isLoading || !user) return <Loading />
+  /* TODO: error handling */
+
+  return (
+    <div className='flex flex-col gap-6 '>
+      <div className='flex flex-col'>
+        <h2 className='font-semibold'>Icon</h2>
+        <MdAccountCircle
+          className={`text-4xl ${colorMap[user.icon]}`}
+          size={52}
+        />
+      </div>
+      <div>
+        <h2 className='font-semibold'>Name</h2>
+        <p className='text-lg'>{user.name}</p>
+      </div>
+      <div>
+        <h2 className='font-semibold'>Email</h2>
+        <p className='text-lg'>{user.email}</p>
+      </div>
+      {/* <div>
+        <h2 className='font-semibold'>Password</h2>
+        <Link href='/edit-password'>
+          <a className='text-lg'>Change Password</a>
+        </Link>
+      </div> */}
```

Comment:
Could you add TODO comment too?
`TODO: Add functionality for changing password`

And I think if you turn on the comment, it shows an error. You need to delete a tag inside of Link. Anyway, you can actually use `RouterLink` component!

Reply:
Got it! Fixed. 
5859f19604f9540bc5d483a2b325af893ce0a533